src/Goerli-DssSpell.t.base.sol

```diff
@@ -463,7 +522,7 @@ contract GoerliDssSpellTestBase is Config, DSTest, DSMath {
                 assertTrue(mat >= RAY && mat <= 300 * RAY, concat("TestError/vat-mat-range-", ilk));  // cr eq 100% and lt 30000%
             } else {
                 assertEq(mat, normalizedTestMat, concat("TestError/vat-mat-", ilk));
-                assertTrue(mat >= RAY && mat < 10 * RAY, concat("TestError/vat-mat-range-", ilk));    // cr eq 100% and lt 1000%
+                assertTrue(mat >= 0 && mat < 10 * RAY, concat("TestError/vat-mat-range-", ilk));    // cr gt 0% and lt 1000%
```

Comment:
The `mat >= RAY` check seems like something we still want to test for other classes. Can we still check that for classes different than 4?
Also, not sure yet if we have it elsewhere, but we should validate there is no pip configured in the spot for class 4 collaterals.

Reply:
Yeah I was about to comment this, I think we should do a special case for this one as well.

Reply:
Ok I think I'll skip this check in the case the spotter has no pip defined.